Add a community software section and move some stuff inside (#220)
CleverCloud/documentation: content/doc/administrate/log-management.md

{{< callout type="warning" >}}
NewRelic has two zones, **EU** and **US**. An account on one zone is not available on the other, make sure to target the right intake endpoint (`log-api.eu.newrelic.com` or `log-api.newrelic.com`).
{{< /callout >}}

### Community software

{{< callout type="info">}}

> Community software is not directly supported by Clever Cloud. It is developed by our community. We do not garantee they will be free of problems.

Reviewer: ```suggestion
Community software isn't directly supported by Clever Cloud. It's developed by our community. We don't  guarantee their maintenance or correct functioning.
```

> You had better open issues on their Git repositories instead of contacting our support.

Reviewer: ```suggestion
You are better off opening issues on their GitHub repositories than contacting Clever Cloud support.
```
